lib/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -19,9 +19,14 @@ Changes
 =======
 
 * Abstracted several new functions in the glue layer to improve compatibility on top of the master branch.
-* Re-organized NVS keys usage to make it range-bound (0xCA00, 0xCAFF); this range is not backward compatible, so you should not rely on pre-existing information saved in flash by earlier versions of this library.
-* Added APIs to set the following values from the application. The application should set and maintain these values to reflect the state of the device. Updated values are pushed to the servers autonomously.
-	* Available Power Sources, Power Source Voltage, Power Source Current, Battery Level, Battery Status, Memory Free, Memory Total, Error Code.
+* Reorganized NVS keys usage to make it range-bound (0xCA00, 0xCAFF).
+  This range is not backward compatible, so you should not rely on pre-existing information saved in flash by earlier versions of this library.
+* Added APIs to set the following values from the application.
+  The application must set and maintain these values to reflect the state of the device.
+  Updated values are pushed to the servers autonomously.
+
+    * Available Power Sources, Power Source Voltage, Power Source Current, Battery Level, Battery Status, Memory Free, Memory Total, Error Code.
```

Comment:
There is no API to push the value of the `Memory Free` resource, seems like we have mistakenly included it in the changelog and so it should be removed.

Reply:
'memory free' removed from both files